### DISCUSSION TITLE
Correct way to transfer Wordpress site to coolify

### DISCUSSION THREAD

**Discussion by chz** [coollabsio/coolify#759] [Q&A] [answered]
Is any one know correct way to transfer site to coolify. Data + mysql

**Comment by vasani-arpit** [comment]
1. Create DB in coolify
2. Deploy backend code in coolify
3. Connect those two and make sure everything works
4. Create db data dump of previous instance
5. populate the dump into new coolify DB

**Reply by tutods** [reply]
I only have files, like the database and the wordpress files, but I can't restore. Can give me a help?

**Comment by chz** [comment]
website is wordpress

**Reply by dualizeo** [reply, marked as answer]
You can install a backup plugin in your current WordPress website and back it up using plugin,
then on coolify you make a new Service ( WordPress website ) and install the same backup plugin and upload your backup from previous website to your new coolify WordPress website you made so you can restore from backup. 
https://wordpress.org/plugins/wpvivid-backuprestore/

**Comment by rockypod** [comment]
I used the [All in one Migration plugin](https://wordpress.org/plugins/all-in-one-wp-migration/) today. Worked like a charm.

It may be faster to keep the backup on your original server. Then fetch it from the Docker container.
